src/algebra/group/with_one.lean

```diff
@@ -27,6 +27,12 @@ def with_one (α) := option α
 
 namespace with_one
 
+instance [has_repr α] : has_repr (with_zero α) :=
+⟨λ o, match o with | none := "0" | (some a) := "(some " ++ repr a ++ ")" end⟩
+
+instance [has_repr α] : has_repr (with_one α) :=
```

Comment:
```suggestion
@[to_additive]
instance [has_repr α] : has_repr (with_one α) :=
```

Reply:
Are you sure that works? I thought `to_additive` wouldn't look in the string.

Reply:
You're right, it doesn't:
```
#eval (0 : with_zero ℕ)
-- 1
```

Reply:
You should maybe still have this attribute to connect the manual 0 instance to the manual one instance, although it's unlikely to matter much